external-crates/move/move-command-line-common/src/character_sets.rs

```diff
@@ -33,21 +35,34 @@ pub fn is_permitted_newline_crlf_chars(c1: char, c2: char) -> bool {
     is_cr && is_lf
 }
 
+// Taken from https://en.wikipedia.org/wiki/Bidirectional_text
+// TODO Double check 200F 200E 061C
```

Comment:
Rust does not FWIW. I think these change the left right behavior because they add accents/character modifiers backwards?

Reply:
That's my understanding from reading up on them. I'm fine with us not banning them, but since there is any question in my mind I default to "ban it" and then we can relax it later if we need to.

Reply:
I understand the concern, but I really don't want to hinder specific languages. So I think they should be fine. And hopefully I don't regret this ;)